services/src/main/java/org/keycloak/services/resources/RealmsResource.java

```diff
@@ -183,4 +189,31 @@ public Response getWellKnown(final @PathParam("realm") String name,
         return Response.ok(wellKnown.getConfig()).cacheControl(CacheControlUtil.getDefaultCacheControl()).build();
     }
 
+    /**
```

Comment:
Please remove adding custom REST endpoints. It's beyond the scope of this work and we have someone else working on it already.